api/proto/teleport/legacy/types/types.proto

```diff
@@ -1678,7 +1678,7 @@ message AuthPreferenceSpecV2 {
 
 // AssistOptions is a set of options related to the Teleport Assist feature.
 message AssistOptions {
-  string APIURL = 1 [(gogoproto.jsontag) = "api_url,omitempty"];
+  string OpenAIToken = 1 [(gogoproto.jsontag) = "openai_token,omitempty"];
```

Comment:
This is probably nit-picking, but I'd rename this to `api_token` or create a hierarchy
```yaml
assist:
  openai:
    api_token: 123
```
Otherwise, if we ever decide to use a different backend we will have this one field that is kind of place.